.github/workflows/column2Vec_test.yml

```diff
@@ -0,0 +1,31 @@
+name: "Column2Vec tests"
+
+on:
+  schedule:
+    - cron: '0 10 * * 6'
```

Comment:
Why the schedule? Also why not on PR?

Reply:
It takes one hour to run.

Reply:
I have a feeling that goes over the limits of free GH Actions. Please verify

Reply:
But in case of a long run maybe we want to do it on `push: maste`

Reply:
It should be 2,000 mins per month [here](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)

I can rewrite it to `push: master`, but that could be more than once per week.